lib/ILDAPWrapper.php

```diff
@@ -166,7 +166,7 @@ public function read($link, $baseDN, $filter, $attr);
 
 	/**
 	 * Search LDAP tree
-	 * @param resource $link LDAP link resource
+	 * @param resource|resource[] $link LDAP link resource
```

Comment:
@jvillafanez is it really capable to handle array of resources? As I see calling it will end up in `invokeLDAPMethod` that will execute `ldap_read()` using `resource[]` as the first argument

Reply:
fixed